package.json

```diff
@@ -19,9 +19,12 @@
 		"type": "git",
 		"url": "https://github.com:woocommerce/woocommerce-gutenberg-products-block.git"
 	},
+	"config": {
+		"WOOCOMMERCE_BLOCKS_PHASE": "experimental"
+	},
```

Comment:
This works for Gutenberg because the [WordPress core package.json](https://github.com/WordPress/wordpress-develop/blob/master/package.json) doesn't define this configuration so the default in the Gutenberg config falls back to `1`.  The blocks feature plugin is included in WooCommerce core differently via composer pulling in a tagged release that is _already built_.  So effectively what you've implemented here means that `experimental` is _always_ used (I verified by doing `cross env WOOCOMMERCE_BLOCKS_PHASE=stable npm run build`).

Instead, what you want to do is remove this configuration and _only_ configure the feature gating via the environment variable.  The `DefinePlugin` definition would have to be something like this instead:

```js
new DefinePlugin( {
				// Inject the `WOOCOMMERCE_BLOCKS_PHASE` global, used for feature flagging.
				'process.env.WOOCOMMERCE_BLOCKS_PHASE': JSON.stringify(
					process.env.WOOCOMMERCE_BLOCKS_PHASE ||
						'experimental'
				),
			} ),
```